setup.py

```diff
@@ -51,25 +51,25 @@
                                   "psyclone.tests.*"])
 
 NAME = 'PSyclone'
-AUTHOR = ('Rupert Ford <rupert.ford@stfc.ac.uk>, '
+AUTHOR = ('Rupert Ford, '
           'Andrew Porter <andrew.porter@stfc.ac.uk>, '
           'Sergi Siso <sergi.siso@stfc.ac.uk>')
 AUTHOR_EMAIL = 'andrew.porter@stfc.ac.uk'
 URL = 'https://github.com/stfc/psyclone'
 DOWNLOAD_URL = 'https://github.com/stfc/psyclone'
-DESCRIPTION = ('PSyclone - a compiler for Finite Element/Volume/Difference'
-               ' DSLs in Fortran')
+DESCRIPTION = ('PSyclone - a source-to-source Fortran compiler for HPC '
+               'applications')
```

Comment:
TBH, I would prefer to keep the DSL aspect visible here. What about:
> PSyclone - a source-to-source Fortran compiler for HPC applications with domain-specific language extensions'

Reply:
Good idea. I've changed to "...source-to-source and DSL Fortran compiler..." to try to make it clear we don't just do DSLs.